Models/Interior/control-lock/control-lock.xml

```diff
@@ -59,57 +59,29 @@
             <button>0</button>
             <repeatable>false</repeatable>
             <binding>
-                <condition>
-                    <greater-than>
-                        <property>sim/model/c172p/cockpit/yoke-aileron</property>
-                        <value>-0.05</value>
-                    </greater-than>
-                    <less-than>
-                        <property>sim/model/c172p/cockpit/yoke-aileron</property>
-                        <value>0.05</value>
-                    </less-than>
-                    <greater-than>
-                        <property>sim/model/c172p/cockpit/yoke-elevator</property>
-                        <value>-0.05</value>
-                    </greater-than>
-                    <less-than>
-                        <property>sim/model/c172p/cockpit/yoke-elevator</property>
-                        <value>0.05</value>
-                    </less-than>
-                    <not>
-                        <property>sim/model/hide-yoke</property>
-                    </not>
-                </condition>
                 <command>property-assign</command>
                 <property>sim/model/c172p/cockpit/control-lock-placed</property>
                 <value>true</value>
             </binding>
             <binding>
-                <condition>
-                    <or>
-                        <less-than-equals>
-                            <property>sim/model/c172p/cockpit/yoke-aileron</property>
-                            <value>-0.05</value>
-                        </less-than-equals>
-                        <greater-than-equals>
-                            <property>sim/model/c172p/cockpit/yoke-aileron</property>
-                            <value>0.05</value>
-                        </greater-than-equals>
-                        <less-than-equals>
-                            <property>sim/model/c172p/cockpit/yoke-elevator</property>
-                            <value>-0.05</value>
-                        </less-than-equals>
-                        <greater-than-equals>
-                            <property>sim/model/c172p/cockpit/yoke-elevator</property>
-                            <value>0.05</value>
-                        </greater-than-equals>
-                    </or>
-                    <not>
-                        <property>sim/model/hide-yoke</property>
-                    </not>
-                </condition>
-                <command>nasal</command>
-                <script>screen.log.write("You have to center aileron and elevator before placing the control lock!", 1, 0.5, 0);</script>
+                <command>property-assign</command>
+                <property>sim/model/c172p/cockpit/yoke-aileron</property>
```

Comment:
What is `/sim/model/c172p/cockpit/{yoke-aileron,yoke-elevator}` exactly? If you set the `/controls/flight/{aileron,elevator}` to 0 (like you do below), the yoke and control surfaces will both go to the center position. So I don't understand why you (additionally) set `/sim/model/c172p/cockpit/{yoke-aileron,yoke-elevator}`?